libs/ui/src/lib/tabs/tab-group.component.html

```diff
@@ -13,7 +13,14 @@
           (keyup.arrowright)="makeActiveNext($event)"
           [attr.aria-controls]="baseId + '-tabpanel-' + i"
           class="cvi-tab-group__tab-btn">
-    {{ tab.title }}
+    <cvi-ng-labeled-icon
```

Comment:
1. It's better to replace this component with just a combination of `cvi-ng-track` and `cvi-ng-icon`. This `cvi-ng-labeled-icon` is a rare and unfortunate case of a component with a specific usage being used too universally... Need to be deprecated or something in future
2. Once replaced – both `cvi-ng-track` and `cvi-ng-icon` need not to appear if the icon is not provided. At the moment the track and empty icon cause this empty space if an icon is not added: 
![image](https://github.com/e-gov/cvi/assets/1833029/16a8f1fd-1bd0-43c1-8f6a-17ed1be726be)